__tests__/pathFinding.test.ts

```diff
@@ -68,6 +68,30 @@ describe('test path finding', () => {
                 { x: 4, y: 4 },
             ],
         },
+        {
```

Comment:
If it grows a bit more, maybe we can split this "data" from the Data Driven Test into a separated file